superduperdb/backends/local/artifacts.py

```diff
@@ -69,6 +70,29 @@ def _load_bytes(self, file_id: str) -> bytes:
         with open(os.path.join(self.conn, file_id), 'rb') as f:
             return f.read()
 
+    def _save_file(self, file_path: str, file_id: str):
+        """
+        Save file in artifact store and return the relative path
+        return the relative path {file_id}/{name}
+        """
+        path = Path(file_path)
+        name = path.name
+        file_id_folder = os.path.join(self.conn, file_id)
+        os.makedirs(file_id_folder, exist_ok=True)
+        save_path = os.path.join(file_id_folder, name)
+        logging.info(f"Copying file {file_path} to {save_path}")
+        if path.is_dir():
+            shutil.copytree(file_path, save_path)
```

Comment:
@jieguangzhou @blythed 
Since we are copying it in artifact as a snapshot of the directory, we need to caution users that if they change the source directory after db.load , it will not be reflected in the component.
although it is intended to be like this

Reply:
These are great points, we should handle all of this through the db.load API.